source/core/src/core/bronze/application/submitted_transactions/unknown_submitted_transactions_handler.py

```diff
@@ -0,0 +1,17 @@
+from pyspark.sql import DataFrame
+
+from core.bronze.domain.constants.column_names.bronze_submitted_transactions_column_names import (
+    ValueColumnNames,
+)
+from core.bronze.infrastructure.repositories.invalid_submitted_transactions_repository import (
+    InvalidSubmittedTransactionsRepository,
+)
+from core.silver.application.protobuf.protobuf_versions import ProtobufVersions
+
+
+def handle(submitted_transactions: DataFrame) -> None:
+    collected_versions = ProtobufVersions().get_versions()
+    unknown_protobuf_messages = submitted_transactions.filter(
+        f"{ValueColumnNames.version} not in ({','.join(collected_versions)})"
+    )
```

Comment:
```suggestion
    collected_versions = ','.join(ProtobufVersions().get_versions())
    unknown_protobuf_messages = submitted_transactions.filter(
        f"{ValueColumnNames.version} not in (collected_versions))"
    )
```

Optional, just thought it made the filter more readable